### PR TITLE
chore(typo): eliminate typos

### PR DESCRIPTION
**Pull request by HansRobo**
## Description

Current `autoware.universe` causes a lot of cspell checker errors but it is left alone.
Some cspell checker errors are caused by not being registered in the checker's dictionary, but many consist of real typos.
This PR aims to eliminate those typos.
The progress of the typo fix is shown at the end of the PR

(note) : PR for adding words to the dictionary is also planned

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by HansRobo** [comment]
@1222-takeshi 
I'm in the process of fixing typos that exists in autoware.universe.
The word `actived` used in the variable name below does not exist, so I would like to modify it. How should I modify it?
(Strictly present but by no means common)
https://github.com/autowarefoundation/autoware.universe/blob/c5e72461e180109adb8356dfc0edc52e2036bb88/perception/ground_segmentation/include/ground_segmentation/gencolors.hpp#L100

Here are my suggestions
- active
- activated

**Comment by yukkysaito** [comment]
I think original code use the word. :cry: 
https://github.com/search?q=activedDists&type=code

**Comment by HansRobo** [comment]
> I think original code use the word. cry https://github.com/search?q=activedDists&type=code

@yukkysaito Thank you for investigation!
In the case the spelling comes from the original code, it is hard to determine if it's a miss spelling or not, so I'm going to embedding a comment out that disables cspell

**Comment by HansRobo** [comment]
@HiroIshida CC: @TakaHoribe @yukkysaito 

Could you come up with a more descriptive variable name instead of `inpro`?
I came up with the possibility that `inpro` means `inner product`, but I'm not sure it's correct.
https://github.com/autowarefoundation/autoware.universe/blob/f984fbb708cb02947ec2824ce041c739c35940f7/planning/freespace_planning_algorithms/src/rrtstar.cpp#L158-L159

Note: `inpro` causes spell-checker errors.

**Comment by yukkysaito** [comment]
@HansRobo 
I'm almost certain it's an inner product from the expression.
image

**Comment by HiroIshida** [comment]
@HansRobo 
Sorry for late response. Yes, it means inner product

**Comment by xmfcx** [comment]
@HansRobo could you rebase this PR to main, it'd be good to merge this.

**Comment by HansRobo** [comment]
> @HansRobo could you rebase this PR to main, it'd be good to merge this.

@xmfcx Thank you for your advice!
I rebased this branch to latest main.

**Comment by xmfcx** [comment]
Also if you are ok with it, can you mark it as ready for review? It already has too many changes, if more are needed, it's better to open another PR.

**Comment by HansRobo** [comment]
You are right.
There are still many typos to fix, but I will finish adding any commits to this PR so as not to increase the burden on reviewers.

**Comment by xmfcx** [comment]
Also next time, it's better to separate fixes on comments and code refactoring. This will also make it easier to review too. Thanks for working on this nevertheless.

**Comment by HansRobo** [comment]
I got it.
Next time, I will try to be reviewer-friendly PR!

File: common/autoware_auto_geometry/include/geometry/bounding_box/lfit.hpp
Comment (yukkysaito) [inline common/autoware_auto_geometry/include/geometry/bounding_box/lfit.hpp:22]: I think LFIT maybe means "L-Shape Fitting"
```suggestion
// LFIT means "L-Shape Fitting"
```

**Comment by yukkysaito** [comment]
@mitsudome-r @kenji-miyake @xmfcx 
I can't check all the details, but I think it's OK to merge once because it may be difficult to avoid conflicts if the merge is delayed. What do you think?

**Comment by xmfcx** [comment]
Agree with @yukkysaito, I will also skim through the changes and review.

File: common/autoware_auto_geometry/include/geometry/bounding_box/lfit.hpp
Comment (HansRobo) [reply common/autoware_auto_geometry/include/geometry/bounding_box/lfit.hpp:22]: Thanks, I wanted to have such an information!

**Review by yukkysaito — APPROVED** [review]
LGTM
Waiting for comment from @kenji-miyake @mitsudome-r ~@xmfcx~

**Comment by kenji-miyake** [comment]
@yukkysaito I've roughly looked over all changes.
It generally looks to be good, but I feel it partly contains some changes that ignore the words that are not recommended to ignore.
Ideally, it's better to track them vis an issue.

Also, actually, I don't like the idea to merge large PRs without enough checks, out of concern for conflicts. Instead, we should separate them into small PRs if there are no dependencies.
In this case, in my opinion, it was better if this PR was split into the following PRs:

- Changing only comments or docs.
  - Just fixing typos.
  - Ignoring several words.
- Changing code.

But it's okay for me to merge it because it's not a very big issue.

**Review by kenji-miyake — APPROVED** [review]

**Comment by yukkysaito** [comment]
>Also, actually, I don't like the idea to merge large PRs without enough checks, out of concern for conflicts. Instead, we should separate them into small PRs if there are no dependencies.
In this case, in my opinion, it was better if this PR was split into the following PRs:
Changing only comments or docs.
Just fixing typos.
Ignoring several words.
Changing code.
But it's okay for me to merge it because it's not a very big issue.

I think so too
(There is same comment from @xmfcx )
https://github.com/autowarefoundation/autoware.universe/pull/2216#issuecomment-1420858590

**Comment by HansRobo** [comment]
>  I feel it partly contains some changes that ignore the words that are not recommended to ignore.

In some cases, I ignored technical or internal terms used in the code (which are difficult for outsiders to understand) intentionally.

The words can be tracked with cspell comments, so I think we can fix it later.
and, the cspell comments is not beautiful, so I hope that contributors who know the meaning of the term will correct it.

Now, that most cspell errors are missed, I believe that cspell comments may appeal more to contributors about the inappropriateness of the terminology.

**Review by taikitanaka3 — APPROVED** [review]

**Comment by HansRobo** [comment]
Sorry, I accidentally made a mistake in git operation, so I reverted.
This resets the code to when it was reviewed.

**Review by yukke42 — APPROVED** [review]

**Review by manato — APPROVED** [review]
@HansRobo 
LGTM regarding files I'm in charge of: 
- common/tensorrt_common/src/tensorrt_common.cpp
- perception/tensorrt_yolox/README.md
- perception/tensorrt_yolox/include/tensorrt_yolox/tensorrt_yolox.hpp
- perception/tensorrt_yolox/src/tensorrt_yolox.cpp

Really appreciate your suggestion!

**Review by angry-crab — APPROVED** [review]
LGTM for `lidar_centerpoint_tvm`

**Review by KYabuuchi — APPROVED** [review]
LGTM for `localization/ndt_scan_matcher` :+1:

**Review by badai-nguyen — APPROVED** [review]
LGTM for 

- sensing/image_diagnostics/README.md
- sensing/pointcloud_preprocessor/docs/blockage_diag.md
- sensing/pointcloud_preprocessor/docs/image/blockage_diag_flowchart.drawio.svg
- sensing/pointcloud_preprocessor/src/outlier_filter/dual_return_outlier_filter_nodelet.cpp

Thank you for your PR!

**Review by ktro2828 — APPROVED** [review]

**Comment by HansRobo** [comment]
Sorry for repeated rebase & force-push

**Comment by HansRobo** [comment]
@xmfcx @mitsudome-r CC: @kenji-miyake 
Can you reviw as a global code owner?
It is unrealistic to ask reviews quickly from all reviewers.

File: planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp
Comment (maxime-clem) [inline planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp:257]: This can be removed.

File: planning/behavior_path_planner/config/avoidance/avoidance.param.yaml
Comment (maxime-clem) [inline planning/behavior_path_planner/config/avoidance/avoidance.param.yaml:21]: This can be removed (the parameter is already defined l. 12).
This seems to have been introduced by https://github.com/autowarefoundation/autoware.universe/commit/026a4b40a29d4e96a8d3792f0575efe08f25798a

**Review by maxime-clem — CHANGES_REQUESTED** [review]
I found a small issue that needs to be solved before merging this PR.

File: planning/behavior_path_planner/config/avoidance/avoidance.param.yaml
Comment (HansRobo) [reply planning/behavior_path_planner/config/avoidance/avoidance.param.yaml:21]: Thank you for pointing it out!
I'll check main branch and fix it

Comment (HansRobo) [reply planning/behavior_path_planner/config/avoidance/avoidance.param.yaml:21]:
```suggestion
```

File: planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp
Comment (HansRobo) [reply planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp:257]:
```suggestion
```

**Review by maxime-clem — APPROVED** [review]

**Review by asana17 — APPROVED** [review]

File: system/system_monitor/launch/system_monitor.launch.xml
Comment (kenji-miyake) [inline system/system_monitor/launch/system_monitor.launch.xml:9]: @HansRobo @takayuki5168 Is this okay? Not typo fixes.

Comment (takayuki5168) [reply system/system_monitor/launch/system_monitor.launch.xml:9]: Not okay

Comment (HansRobo) [inline system/system_monitor/launch/system_monitor.launch.xml:9]:
```suggestion
  <arg name="cpu_monitor_config_file" default="$(find-pkg-share system_monitor)/config/cpu_monitor.param.yaml"/>
  <arg name="hdd_monitor_config_file" default="$(find-pkg-share system_monitor)/config/hdd_monitor.param.yaml"/>
  <arg name="mem_monitor_config_file" default="$(find-pkg-share system_monitor)/config/mem_monitor.param.yaml"/>
  <arg name="net_monitor_config_file" default="$(find-pkg-share system_monitor)/config/net_monitor.param.yaml"/>
  <arg name="ntp_monitor_config_file" default="$(find-pkg-share system_monitor)/config/ntp_monitor.param.yaml"/>
  <arg name="process_monitor_config_file" default="$(find-pkg-share system_monitor)/config/process_monitor.param.yaml"/>
  <arg name="gpu_monitor_config_file" default="$(find-pkg-share system_monitor)/config/gpu_monitor.param.yaml"/>
  <arg name="voltage_monitor_config_file" default="$(find-pkg-share system_monitor)/config/voltage_monitor.param.yaml"/>
```

Comment (HansRobo) [reply system/system_monitor/launch/system_monitor.launch.xml]: Reverted in https://github.com/autowarefoundation/autoware.universe/pull/2216/commits/1ac191a1c26238094a8913aa41aa3cf31eac21a1

Comment (takayuki5168) [reply system/system_monitor/launch/system_monitor.launch.xml]: Could you check if scenario simulator (or planning simulator) works well with this PR in order to make sure that this kind of mistakes do not exist.

**Comment by kenji-miyake** [comment]
I'll merge this after CI has passed.
@HansRobo has double-checked the diff. Thank you for your work!

**Comment by HansRobo** [comment]
waiting for Autoware Evaluator

**Pull request merged**
